Organiser.Core/Organiser.Core/Models/SQL/Bugs.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE Bugs (
+	BID INT IDENTITY(1,1) NOT NULL PRIMARY KEY,
```

Comment:
all column names here in this table definition are prefixed with B character. I guess the reason was to emphasize belonging to Bugs table. but I think it is redundant information and complicates. 

second thing - why you need this SQL when you have entity framework and migrations?